### PR TITLE
Fix Magazine and Article Notifications + Create Prod and Dev Magazine Google Forms

### PR DESCRIPTION
**Pull request by kidzegeye**
## Overview

* Fix Article and Magazine creation notifications
* Separate the magazine creation form between development and production versions (two Google forms)

## Changes Made

* Fix Article and Magazine creation notifications
* Separate the magazine creation form between development and production versions (two Google forms)
* Modified references in the backing Google sheet for each form

## Test Coverage

Tested on test magazines sent to the DEV Google form. After adding the magazine, ran the microservice and inspected if the notifications were sent

### CODE REVIEW

**Review by Archit404Error — APPROVED** [review]
Good to go

File: src/app.py [hunk 1]
```diff
@@ -78,13 +88,13 @@ def gather_magazines():
         len_data = len(data)
         parse_counter = 1
         for i in range(1, len_data):
-            if data[i][5] != "1" and data[i][0] != "":
```
Comment (chalo2000) [inline src/app.py:81]: some more comments here would make this easier for someone to edit this in the future if we want to improve on our system

File: src/app.py [hunk 2]
```diff
@@ -22,10 +26,15 @@
 DATABASE = os.getenv("DATABASE")
 VOLUME_NOTIFICATIONS_ENDPOINT = os.getenv("VOLUME_NOTIFICATIONS_ENDPOINT")
 GOOGLE_SERVICE_ACCOUNT_PATH = os.getenv("GOOGLE_SERVICE_ACCOUNT_PATH")
+SERVER = os.getenv("SERVER")
 
+if SERVER == "prod":
```
Comment (chalo2000) [inline src/app.py:31]: why is it that the GOOGLE_SHEET_IDs are in the codebase and not in an environments file? could they be abused?

File: src/magazine.py [hunk 3]
```diff
@@ -19,13 +19,12 @@ class Magazine:
     def __init__(self, sheet_row, publication):
         pf.load_censor_words(FILTERED_WORDS)
         self.timestamp = sheet_row[0]
-        self.slug = sheet_row[1]
-        self.title = sheet_row[2]
-        self.drive_link = sheet_row[3]
+        self.slug = sheet_row[2]
```
Comment (chalo2000) [inline src/magazine.py:22]: this is a lot more understandable than the gather_magazines() function

**Review by chalo2000 — APPROVED** [review]
I don't understand why there are commits in here from almost a year ago; is this something that wasn't merged a while back? Also left some comments about code clarity and questions about environment variable choices.

**Pull request merged**